frontend/src/components/widgets/CardBadges.js

```diff
@@ -98,17 +102,24 @@ export default ({ card }) => {
 
       {openPrCount ? (
         <Tooltip title="Number of open pull requests on this card and their age">
-          <Badge badgeContent={openPrCount} color="primary"> 
-            {oldestOpenPrUpdatedTime === null ? (
-              <CallMergeIcon />
-            ) : (
-              <Chip
-              avatar={<CallMergeIcon />}
-              className={classes.chip}
-              label={getAgeString(oldestOpenPrUpdatedTime)}
-            />
-            )}
-          </Badge>
+          <Button
```

Comment:
Do you really want to make a button act as a LInk ?
Would you perhaps consider using an anchor tag

Reply:
Anchor tag used instead of a button 

Since the Link component is built on top of the Typography component, It ends up requiring a lot of styling just to retain the original position of the chip and badge.